### ISSUE TITLE
What's the difference between BPC and cross-entropy loss?

### ISSUE THREAD

**Issue by waltonfuture** [hkust-nlp/llm-compression-intelligence#1]
↑

**Comment by HYZ17** [comment]
For a given sequence x, there is a mathematical relationship between the total cross entropy loss and the BPC (Bits Per Character), which is a constant coefficient difference and does not affect linear regression. However, for the loss per token, since different models have different tokenizers, and the information contained in each token varies, the loss per token is not directly comparable.

**Comment by jxhe** [comment]
To add based on the discussion on X, different denominators on the cross-entropy loss due to varying tokenizers do make a big difference.
